docs/concepts/metric-exporter.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Metric Exporter"
+linkTitle: Metric Exporter
+---
+## Overview
+
+Following the [exporter pattern](https://prometheus.io/docs/instrumenting/exporters/), we maintain and distribute our own [Prometheus](https://prometheus.io/) exporter for monitoring & alerting of Ondat volumes. The metrics our exporter publishes include data on volume health, capacity & traffic.
+
+Our exporter’s source code can be found [here](https://github.com/ondat/metrics-exporter).
+
+Please see our [operations page](/docs/operations/metric-exporter/) to get started.
+
+Additionally, we distribute Grafana dashboards to visualize the data. They can be found [here](https://github.com/ondat/metrics-exporter/tree/main/grafana).
+
+Likewise, we distribute example alerting rules for our metrics using [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/), they can be found [here](https://github.com/ondat/metrics-exporter/tree/main/alertmanager)
+
+Contributions to both are welcome!
+
+> ⚠️ When installing be sure to create the rules in the same namespace as your Prometheus resource and match its ruleSelector field.
```

Comment:
We tell the customer to match the `ruleSelector field` and I think most people will not make any sense out of this.
I initially thought this and the next file on this PR `docs/operations/metric-exporter.md` would be together thus the user could look at the example there but as Cannis explained they won't so we need to be more clear.

I suggest we are a bit more clear about what we mean here or have a link to the other page.

If you prefer the first option, I would consider something like to be good enough:
```
> ⚠️ When setting up a [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/1e4acb010642067bb918eebb75410191640a95c6/Documentation/user-guides/getting-started.md) be sure to create the rules in the same namespace as your Prometheus resource and have its `selector` field match the labels of ours services exposing metrics (see example manifest in <link to the other page - don't know how to do it here>
```

Reply:
We can add a link

Reply:
updated